lib/game.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+require_relative 'player.rb'
+require_relative 'board.rb'
+
+# Game class
+class Game
+  attr_reader :board_game, :player1, :player2
+  attr_accessor :symbol1, :symbol2
+
+  def initialize
+    @input = PlayerInput.new
+    @board_game = Board.new
+    @current_player = @player1
+    @symbol1 = 'X'
+    @symbol2 = 'O'
+    @end_round = false
+  end
+
+  def players(first, second)
+    @player1 = Player.new(first, @symbol1)
+    @player2 = Player.new(second, @symbol2)
+  end
+
+  def change_player
+    @current_player = @current_player == @player1 ? @player2 : @player1
+  end
+
+  def player_message
+    if @current_player == @player1
+      puts "#{@player1.player} Choose a number between 1 and 9 to put your symbol."
+    else
+      puts "#{@player2.player} Choose a number between 1 and 9 to put your symbol."
```

Comment:
- [ ] User interaction commands should be in one executable file main.rb (no “puts” and “get.chomp” there!).